src/infrastructure/fonts.js

```diff
@@ -1,3 +1,27 @@
 export const fonts = {
 
+  heading: 'OpenSans_700Bold',
+  subHeading: 'OpenSans_600SemiBold',
+  body: 'Montserrat_400Regular',
+  bodyBold: 'Montserrat_700Bold',
+  light: 'Montserrat_300Light'
+
+};
+
+export const fontWeights = {
+  regular: 400,
+  medium: 500,
+  bold: 700,
+};
+
+export const fontSizes = {
+  caption: '12px',
+  button: '14px',
+  body: '16px',
+  title: '20px',
+  h5: '24px',
+  h4: '34px',
+  h3: '45px',
+  h2: '56px',
+  h1: '112px',
```

Comment:
Do we need these tags here? If not, can you remove them please?

Reply:
no..done.